compiler/src/org.graalvm.compiler.truffle.options/src/org/graalvm/compiler/truffle/options/PolyglotCompilerOptions.java

```diff
@@ -398,6 +398,9 @@ private String indent(int nameLength) {
     @Option(help = "Perform a set of optimizations on each explored call target during inlining.", category = OptionCategory.INTERNAL)
     public static final OptionKey<Boolean> InliningTruffleTierOnExpand = new OptionKey<>(true);
 
+    @Option(help = "Exclude a specific method from being inlined.", category = OptionCategory.INTERNAL)
+    public static final OptionKey<String> ExcludeInlining = new OptionKey<>("");
```

Comment:
I recall that in the issue we discussed having `ExcludeInline` use a similar syntax to `CompileOnly` (https://github.com/oracle/graal/issues/3492#issuecomment-867026345) i.e. a comma separated list of things to exclude.

Reply:
I totally misunderstood the initial syntax comment - thanks for clarifying. Fixed though! https://github.com/oracle/graal/pull/3524/files#diff-b38f98833b5f5f1fed1b5fbf826cd34dd03990f2643a70375033422c6460878eR402